src/org/traccar/protocol/OsmAndProtocolDecoder.java

```diff
@@ -128,6 +128,9 @@ protected Object decode(
                 case "batt":
                     position.set(Position.KEY_BATTERY_LEVEL, Double.parseDouble(value));
                     break;
+                case "driverUniqueId":
```

Comment:
Sorry, I just realized that we don't need this in OsmAnd. Decoder automatically saves all extra parameters in attributes.